.travis.yml

```diff
@@ -16,14 +17,11 @@ addons:
 
 env:
   global:
-  - VERSION="10.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
-  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - VERSION="10.0" TESTS="0" LINT_CHECK="0"
   - WKHTMLTOPDF_VERSION="0.12.4"
```

Comment:
let's move it to 0.12.5 as suggested https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L36